packages/unenv-preset/package.json

```diff
@@ -47,15 +47,15 @@
 	},
 	"devDependencies": {
 		"@types/node-unenv": "npm:@types/node@^22.10.5",
-		"typescript": "catalog:default",
+		"typescript": "^5.7.2",
 		"unbuild": "^3.2.0",
 		"undici": "catalog:default",
+		"unenv": "file:/Users/vberchet/code/work/unenv",
```

Comment:
Really not sure about effect of this on resolution but it won't be same as normal installations in `node_modules`.

We should adopt it with next rc (or nightly if you have it still working here)

Reply:
Yep, that was just pushing the changes that I have for (p)review.
This PR is in draft mode, I just have been more clear in the description.